app/controllers/administrator_controller.rb

```diff
@@ -3,6 +3,8 @@ class AdministratorController < ApplicationController
   def index
     authorize! :index, :administrator
     @users = User.all
+    @characteristics = Characteristic.all
+    @CharacteristicGroups = CharacteristicGroup.paginate(:page => params[:char_groups_page], :per_page => 12)
```

Comment:
@CharacteristicGroups should be @characteristic_groups according to good ruby style. CamelCase variables are always reserved for class names.